### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by renner**
This PR comes with new and more comprehensive README file contents.

### CODE REVIEW

**Comment by aaannz** [comment]
LGTM, what I'm missing is the default/recommended packaging guide. Either in this readme or create a template formula spec.

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-This repository is for Salt formulas.
+# SUSE Salt Formulas
+
+This repository contains Salt Formulas to be shipped for `openSUSE` and `SUSE Linux Enterprise Server`. The Formulas can be used with stand-alone Salt or within [Uyuni](https://github.com/uyuni-project/uyuni) / [SUSE Manager](https://www.suse.com/products/suse-manager/) offering UI support for configuring the states.
```
Comment (mseidl) [inline README.md]: It'd sound better, maybe more accurate instead of saying "configuring the states" -> changing pillar data or configuring the formulas?

**Comment by renner** [comment]
> LGTM, what I'm missing is the default/recommended packaging guide. Either in this readme or create a template formula spec.

Thanks, I will add a `.spec` file template either as a file in the repo to copy from, or in the wiki and link it from here.

File: README.md (hunk 1 above)
Comment (renner) [reply README.md]: Thanks, changed the text slightly.

**Pull request merged**